### PR TITLE
Split `Score` into game mode-specific articles

### PR DESCRIPTION
**Pull request by TicClick**
part of https://github.com/ppy/osu-wiki/issues/6431. nothing meaningful is added/updated/fixed, this will be done in subsequent PRs

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review) (@DoryanR)

### CODE REVIEW

**Comment by DoryanR** [comment]
I'll go see tonight for the French part

**Comment by geshii** [comment]
> I'll go see tonight for the French part. If it's just a copy/paste of the article it will be fine for now, but it will have to be updated, it's planned

It looks good for the French part I guess, I don't see anything to change

File: wiki/Score/osu!/fr.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+no_native_review: true
+outdated: true
+outdated_since: c78e8f94260067c49d36a55deaaf7f40cb796b39
+---
+
+# osu! scoring system
```
Comment (DoryanR) [inline wiki/Score/osu!/fr.md:7]:
```suggestion
# Système de notation d'osu!
```

File: wiki/Score/osu!/fr.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+| ![](/wiki/Skinning/Interface/img/hit50.png "50") | 50 | Score obtenu lorsqu'un cercle est frappé avec du délai, mais assez faible que pour ne pas causer de Miss, lorsqu'un Slider est complété avec beaucoup de ticks ratés, ou lorsqu'un Spinner est complété avec sa jauge modérément remplie. Un score de 50 apparaît avec une couleur orange par défaut. Obtenir un 50 dans un combo empêche l'obtention d'un Katu ou d'un Geki à la fin de celui-ci. |
+| ![](/wiki/Skinning/Interface/img/hit0.png "Miss") | Miss | Score obtenu lorsqu'un cercle n'est pas frappé, ou frappé avec trop de délai, lorsqu'un Slider n'est pas frappé ou maintenu sur au moins un tick, ou lorsqu'un Spinner est complété avec sa jauge peu remplie. Obtenir un Miss réinitialise le combo actuel, et empêche l'obtention d'un Katu ou d'un Geki à la fin de celui-ci. |
+
+## Object score
```
Comment (DoryanR) [inline wiki/Score/osu!/fr.md:20]:
```suggestion
## Score de l'objet
```

File: wiki/Score/osu!catch/fr.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
+---
+no_native_review: true
+outdated: true
+outdated_since: c78e8f94260067c49d36a55deaaf7f40cb796b39
+---
+
+# osu!catch scoring system
```
Comment (DoryanR) [inline wiki/Score/osu!catch/fr.md:7]:
```suggestion
# Système de notation d'osu!catch
```

File: wiki/Score/osu!catch/fr.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+| (None) | Miss Droplets | Droplets (Chemin de Slider) ratées. Elles ne sont pas affichées sur l'écran de résultats, mais le sont côté serveur. Le combo ne sera pas brisé, mais cela diminue la précision et fait perdre un des 50 pouvant être obtenus. Cela n'est pas considéré comme un Miss. N'est pas affiché en multijoueur ou sur l'écran de résultats, mais l'est en tant que *Droplet Miss* dans le classement du site. |
+| ![](img/catch-banana.jpg "Banana") | Bananas | Équivalent du Spinner. Ces petites bananes rétrécissent lorsqu'elles s'approchent du Catcher. Il n'est pas obligatoire de les capturer. 1,100pt en cas de capture, ceci venant de la complétion du Spinner dans osu! qui offre 1000pt bonus ainsi que 100pt pour une rotation. |
+| ![](img/catch-0.jpg "Miss") | Miss | Fruit et Juice Drops ratés. Brise le combe. Obtenu lorsque la capture d'un Fruit/Juice Drops est échouée. Aucun score accordé et brise le combo. Visible uniquement sur l'écran de résultats. |
+
+## Object score
```
Comment (DoryanR) [inline wiki/Score/osu!catch/fr.md:22]:
```suggestion
## Score de l'objet
```

File: wiki/Score/osu!mania/fr.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+---
+no_native_review: true
+outdated: true
+outdated_since: c78e8f94260067c49d36a55deaaf7f40cb796b39
+---
+
+# osu!mania scoring system
```
Comment (DoryanR) [inline wiki/Score/osu!mania/fr.md:7]:
```suggestion
# Système de notation d'osu!mania
```

File: wiki/Score/osu!mania/fr.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+| ![](/wiki/Skinning/osu!mania/img/mania-hit300.png "300") | 300 | Frappe ou maintien presque parfait de la note. Le score maximum ne peut être atteint si ce score est obtenu. |
+| ![](/wiki/Skinning/osu!mania/img/mania-hit200.png "200") | 200 | Frappe ou maintien de la note avec un léger délai. Les nouveaux joueurs peuvent obtenir fréquemment ce score jusqu'à ce qu'ils s'habituent aux mécanismes de défilement et de vitesse, aux touches et à la zone de jugement. |
+| ![](/wiki/Skinning/osu!mania/img/mania-hit100.png "100") | 100 | Frappe ou maintien de la note avec un délai modéré. N'affecte pas la jauge de vie. |
+| ![](/wiki/Skinning/osu!mania/img/mania-hit50.png "50") | 50 | Frappe ou maintien de la note avec un délai important. Score par défaut si une note maintenue n'est pas relâchée. Faible réduction de la jauge de vie. Brise le combo. |
+| ![](/wiki/Skinning/osu!mania/img/mania-hit0.png "Miss") | Miss | Ne pas frapper ou maintenir la note, ou utiliser les mauvaises touches. Importante réduction de la jauge de vie et brise le combo. |
+
+## Scoring
```
Comment (DoryanR) [inline wiki/Score/osu!mania/fr.md:20]:
```suggestion
## Notation
```

File: wiki/Score/osu!taiko/fr.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
+---
+no_native_review: true
+outdated: true
+outdated_since: c78e8f94260067c49d36a55deaaf7f40cb796b39
+---
+
+# osu!taiko scoring system
```
Comment (DoryanR) [inline wiki/Score/osu!taiko/fr.md:7]:
```suggestion
# Système de notation d'osu!taiko
```

File: wiki/Score/osu!taiko/fr.md [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+
+## Hit values
+
+| Image | Nom | Effet |
+| :-: | :-: | :-- |
+| ![](/wiki/Skinning/osu!taiko/img/taiko-hit300k.png "GREAT") ![](/wiki/Skinning/osu!taiko/img/taiko-hit300g.png "GREAT") ![](/wiki/Skinning/osu!taiko/img/taiko-hit300.png "GREAT") | (良) Great | Score obtenu lorsque la note est frappée exactement dans le cercle de jugement, avec la bonne couleur. Image: (1 et 2) Grande Note (3) Petite Note |
+| ![](/wiki/Skinning/osu!taiko/img/taiko-hit100k.png "GOOD") ![](/wiki/Skinning/osu!taiko/img/taiko-hit100.png "GOOD") | (可) Good | Score obtenu lorsque la note est frappée légèrement en avance ou en retard, avec la bonne couleur. Image: (1) Grande Note (2) Petite Note. |
+| ![](/wiki/Skinning/osu!taiko/img/taiko-hit0.png "MISS") | (不可) Miss | Score obtenu lorsque la note n'est pas frappée à temps, ou avec la mauvaise couleur. |
+
+## Object score
```
Comment (DoryanR) [inline wiki/Score/osu!taiko/fr.md:17]:
```suggestion
## Score de l'objet
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
Everything is good for the French side, just need to translate the headers

File: wiki/Score/osu!mania/en.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
+---
+needs_cleanup: true
+---
+
+# Score
+
+## Hit values
```
Comment (DoryanR) [inline wiki/Score/osu!mania/en.md:7]:
```suggestion
# osu!mania scoring system

## Hit values
```

File: wiki/Score/osu!taiko/en.md [hunk 10]
```diff
@@ -0,0 +1,57 @@
+---
+needs_cleanup: true
+---
+
+# Score
+
+## Hit values
```
Comment (DoryanR) [inline wiki/Score/osu!taiko/en.md:7]:
```suggestion
# osu!taiko scoring system

## Hit values
```

**Review by DoryanR — CHANGES_REQUESTED** [review]
I feel like there are headers missing in those places, right?

**Comment by geshii** [comment]
> * wiki/Game_mode/osu!/id.md
> * wiki/Game_mode/osu!/tr.md
> * wiki/Game_mode/osu!catch/fr.md
>* wiki/Game_mode/osu!mania/es.md
> * wiki/Game_mode/osu!mania/fr.md
>* wiki/Game_mode/osu!taiko/fr.md
>
>If your changes DON'T NEED to be added to the translations, add SKIP_OUTDATED_CHECK anywhere in the description of your pull request.

Do we need to edit these files aswell?

**Comment by TicClick** [comment]
yep, I have a local patch for French articles, will find someone for Indonesian/Turkish/Spanish

**Comment by TicClick** [comment]
all done, with help of Zeus-, Sies, Doryan, and Darksonic

**Review by TPGPL — APPROVED** [review]

**Pull request merged**